tests/e2e/specs/in-game/game-over/rematch.spec.js

```diff
@@ -70,16 +72,16 @@ describe('Creating And Updating Ranked Matches With Rematch', () => {
           player1: this.playerOneId,
           player2: this.playerTwoId,
           winner: this.playerOneId,
-          startTime: dayjs().subtract(1, 'week').subtract(1, 'day').valueOf(),
-          endTime: dayjs().subtract(1, 'week').subtract(1, 'day').valueOf(),
+          startTime: dayjs().utc().subtract(1, 'week').subtract(1, 'day').format(),
+          endTime: dayjs().utc().subtract(1, 'week').subtract(1, 'day').format(),
```

Comment:
Why are we using `.utc()` in some places and not others? I'm not clear on whether we want to be using utc() or not.  What's your understanding, @Haviles04 ?

Reply:
I don't believe we want UTC